src/Symfony/Component/Console/EventListener/ErrorListener.php

```diff
@@ -75,19 +75,15 @@ public static function getSubscribedEvents(): array
         ];
     }
 
-    private static function getInputString(ConsoleEvent $event): ?string
+    private static function getInputString(ConsoleEvent $event): string
     {
         $commandName = $event->getCommand()?->getName();
         $input = $event->getInput();
```

Comment:
We can move the cast up.

```suggestion
        $input = (string) $event->getInput();
```

Maybe we should rename the variable to `$inputString` to illustrate that we're not dealing with the input object anymore.